app/javascript/src/components/editor/Editor.svelte

```diff
@@ -92,12 +92,13 @@
       const useNewlines = params.args_length >= $settings["autocomplete-min-parameter-newlines"]
 
       const apply = v.args.map(a => {
-        let string = a.default?.toString().toLowerCase().replaceAll(",", "")
-        if (useParameterObject) string = `${ useNewlines ? "\n\t" : "" }${ a.name }: ${ string }`
+        const name = toCapitalize(a.name?.toString().toLowerCase())
+        let defaultValue = a.default?.toString().toLowerCase().replaceAll(",", "")
+        defaultValue = lowercaseDefaults.includes(defaultValue.toLowerCase()) ?
+          defaults[toCapitalize(defaultValue)] :
+          toCapitalize(defaultValue)
 
-        if (lowercaseDefaults.includes(string)) return defaults[toCapitalize(string)]
-
-        return toCapitalize(string)
+        return useParameterObject ? `${ useNewlines ? "\n\t" : "" } ${ name }: ${ defaultValue }` : defaultValue
```

Comment:
In a similar vain, what do you think of doing this instead?

```suggestion
        if (useParameterObject) return `${ useNewlines ? "\n\t" : "" } ${ name }: ${ defaultValue }` 
        
        return defaultValue
```

Reply:
i feel like if the first return is within the if then the second return should be in the `else`?